core/spec/models/spree/order_spec.rb

```diff
@@ -312,41 +312,6 @@ def compute(_computable)
     end
   end
 
-  context 'empty!' do
```

Comment:
@KacperMekarski let's leave this test until Spree 5

Reply:
@damianlegawiec done!